utils/common.ts

```
@@ -0,0 +1,442 @@
+/**
+ * Converts a byte value into a human-readable file size string with appropriate units (Bytes, KB, MB, GB).
+ *
+ * @param bytes - The file size in bytes
+ * @returns The formatted file size string with units
+ */
+export function formatFileSize(bytes: number): string {
+  if (bytes === 0) return "0 Bytes";
+  const k = 1024;
+  const sizes = ["Bytes", "KB", "MB", "GB"];
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+  return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + " " + sizes[i];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)